### PR TITLE
Admin: functionality implemented

### PR DESCRIPTION
**Pull request by Bourkupine**
Still needs front end work.
Need to add check for user type as admin.
Will add search functionality in a later patch (likely with feedback update).

### CODE REVIEW

File: src/app/login/login_submit.php [hunk 1]
```diff
@@ -18,17 +19,32 @@
...
-    }
+        $_SESSION += get_profile_details($db_con, $user_id);
+
+        if (isset($_COOKIE["first_timer_" . $user_id])) {
+            unset($_COOKIE["first_timer_" . $user_id]);
+            setcookie("first_timer_" . $user_id, '', -1, '/', false);
+            header("Location: ../profile");
+        } else {
+            header("Location: ../dashboard");
+        }
```
Comment (oOoOoOo0o) [inline src/app/login/login_submit.php:47]: Missing a closing brace for the else statement, caused an error

File: src/app/navbar/navbar.php [hunk 2]
```diff
@@ -126,47 +130,59 @@ class="btn-close btn-close-white"
...
-            }
-            ?>">
-              <i class="fa-solid fa-user fa-2x"></i>
-              <span class="navbar-toggler text-white border-0">Profile</span>
-            </a>
-          </li>
-        </ul>
-
-      </div>
+        <div class="offcanva-body">
```
Comment (oOoOoOo0o) [inline src/app/navbar/navbar.php:133]: Should be offcanvas-body

**Review by Blake1412 — APPROVED** [review]

File: src/app/admin/users.php [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+                       name="search-box">
+                <button class="btn" type="submit" name="search-button" id="search-button">Search</button>
+            </div>
+        </form>
+    </div>
+    <!-- User List -->
+    <div class="row">
+        <table class="table table-striped">
+            <?php
+            if (isset($_POST['search-button'])) {
```
Comment (oOoOoOo0o) [inline src/app/admin/users.php:169]: Could be an idea to remove this if and just always load in all users upon entry.
If you do this you will have to add isset($_POST["search-box"]) to line 183

**Review by oOoOoOo0o — CHANGES_REQUESTED** [review]

**Comment by oOoOoOo0o** [comment]
Need to merge with main.
Before banning a user delete all previous bans from user to prevent errors.

**Review by oOoOoOo0o — APPROVED** [review]

**Pull request merged**